app/views/projects/index.html.erb

```diff
@@ -5,7 +5,14 @@
 <%- else %>
 
 <%- current_user.projects.each do |p| %>
-  <div class="project name"><%= link_to p.name, project_path(p) %>, <span><%= link_to 'see merge requests', project_merge_requests_path(p) %></span></div>
+    <div class="project name">
```

Comment:
Better to always have the link, so the user can go there and view with their own eyes that really there's no MR's...

So you can just write the merge_request counter in a helper and use it here. the helper will just return different strings in case of zero or any merge requests.